crates/common/types/account.rs

```diff
@@ -19,11 +19,49 @@ use crate::{
     utils::keccak,
 };
 
-#[allow(unused)]
+#[derive(Clone, Default, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Code {
+    pub hash: H256,
+    pub bytecode: Bytes,
+    pub jump_targets: Vec<u16>,
```

Comment:
It may or may not be worth it to make this an `Arc<[u16]>` instead. That would make cloning much cheaper, and this is immutable once computed so it should be correct.

Reply:
added a comment for the future